pysat/tests/test_utils_coords.py

```diff
@@ -21,37 +26,37 @@ def teardown(self):
         del self.ref_angles
 
     def test_adjust_cyclic_data_default(self):
-        """ Test adjust_cyclic_data with default range """
+        """ Test adjust_cyclic_data with default range."""
 
         ref_rad = np.radians(self.ref_angles) - np.pi
         ref_angles = coords.adjust_cyclic_data(ref_rad)
 
         assert ref_angles.max() < 2.0 * np.pi
         assert ref_angles.min() >= 0.0
+        return
 
     def test_adjust_cyclic_data_custom(self):
-        """ Test adjust_cyclic_data with a custom range """
+        """ Test adjust_cyclic_data with a custom range."""
 
         ref_angles = coords.adjust_cyclic_data(self.ref_angles,
                                                high=180.0, low=-180.0)
 
         assert ref_angles.max() < 180.0
         assert ref_angles.min() >= -180.0
+        return
 
 
-class TestLonSLT():
+class TestUpdateLon():
+    """Unit tests for the `update_longitude` function."""
     def setup(self):
         """Runs after every method to clean up previous testing."""
```

Comment:
```suggestion
        """Set up the unit test environment."""
```
We alternate between types of descriptions for the setup and teardown.  I'm leaning toward this one